### PR TITLE
Enable cargo-spellcheck for `blockchain/`

### LINKED ISSUE #1744: Enable `cargo-spellcheck` for `blockchain/`
**Issue summary**

Make sure that `cargo spellcheck --cfg ../.config/spellcheck.toml` can run without errors in `blockchain/` For details, see the tracking issue.

**Other information and links**

### PR DESCRIPTION
**Pull request by connormullett**
**Summary of changes**
Changes introduced in this pull request:
- clear all spellcheck errors in `blockchain/` directory

**Reference issue to close (if applicable)**

Closes #1744 

**Other information and links**

### CODE REVIEW

File: Cargo.lock [hunk 1]
```diff
@@ -6204,9 +6204,9 @@ version = "1.0.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "b22a693222d716a9587786f37ac3f6b4faedb5b80c23914e7303ff5a1d8016e9"
 dependencies = [
- "proc-macro2 1.0.40",
- "quote 1.0.20",
- "syn 1.0.98",
+ "proc-macro2 1.0.43",
```
Comment (LesnyRumcajs) [inline Cargo.lock:6207]: Not sure why we need to change this. Seems like the same one from the other PR. If a lockfile changed globally for `main`, the best would be to issue a separate PR.

File: blockchain/blocks/src/ticket.rs [hunk 2]
```diff
@@ -4,7 +4,7 @@
 use forest_crypto::VRFProof;
 use forest_encoding::tuple::*;
 
-/// A Ticket is a marker of a tick of the blockchain's clock.  It is the source
+/// A Ticket is a marker of a tick of the block-chain's clock.  It is the source
```
Comment (LesnyRumcajs) [inline blockchain/blocks/src/ticket.rs:7]: Just `blockchain` is not fine? Or should we add it until we have a better dictionary?

**Review by LesnyRumcajs — CHANGES_REQUESTED** [review]

File: blockchain/blocks/src/ticket.rs (hunk 2 above)
Comment (elmattic) [reply blockchain/blocks/src/ticket.rs:7]: Try to suffix blockchain in our dictionary with `/M`, `blockchain/M`.

Comment (elmattic) [reply blockchain/blocks/src/ticket.rs:7]: > It tells hunspell to also accept the possessive version of the word.

File: Cargo.lock (hunk 1 above)
Comment (connormullett) [reply Cargo.lock:6207]: Not sure exactly what caused the change. If I remember right when I checked out the new branch it checked out into unstaged changes (the cargo.lock was updated with these values) despite main not having this issue?? Not sure if we need a new PR for a 3 line change that I'm not sure where it even came from

File: blockchain/blocks/src/ticket.rs (hunk 2 above)
Comment (connormullett) [reply blockchain/blocks/src/ticket.rs]: :+1:

**Review by LesnyRumcajs — APPROVED** [review]

**Review by elmattic — APPROVED** [review]
